### ISSUE TITLE
InputTextareaModule doesnt exist in PrimeNg 18 anymore

### ISSUE THREAD

**Issue by Logic-Bits** [ngx-formly/ngx-formly#4024]
![grafik](https://github.com/user-attachments/assets/3a56fa58-abb2-4d25-a493-e97aa64a58cf)

- Angular version: 17.3.0
- Formly version: 7.0.0-alpha.0

https://v18.primeng.org/textarea
=> The Module is just called Textarea

**Comment by aitboudad** [comment]
Probably we may need to drop primeng v17 support, we'll provide a fix in the upcoming release.

**Comment by aitboudad** [comment]
> Hi, primeng v18 i assume you are talking about. it works in Primeng 17.

yes v18, it case we cannot support both versions, we'll drop v17 support.

**Comment by Logic-Bits** [comment]
Gotcha, sounds good!

**Comment by aitboudad** [comment]
This issue has been fixed and released as part of [v7.0.0-alpha.1](https://github.com/ngx-formly/ngx-formly/releases/tag/v7.0.0-alpha.1) release.

Please let us know, in case you are still encountering a similar issue/problem.
Thank you!
